### DISCUSSION TITLE
individual farmer never get reward ?

### DISCUSSION THREAD

**Discussion by tianxinya2021** [Chia-Network/chia-blockchain#2145] [General]
If I read this correctly, (from https://github.com/Chia-Network/chia-blockchain/wiki/FAQ):

'Once a plot passes the plot filter it then competes for the best proof of space with every other plot that also passed that plot filter for that signage point. For reasons that aren't super simple to intuit, the only thing each plot is competing on is to have the best proof of space and thus the chances of getting a reward depend on total size of plots on the farm'

After passing the filter, smaller space, or smaller farmer, won't beat bigger ones to get reward?
Is my understanding correct?

If true, why we, smaller farmer, as individual, should contribute chia network by farming?

**Comment by hoffmang9** [comment]
Every 104GiB stands an equal chance to win one of 4608 blocks per day.

**Comment by tianxinya2021** [comment]
'chances of getting a reward depend on total size of plots on the farm'?
total size of plots? not each plot size?
which is true then? confusing?

**Reply by hoffmang9** [reply]
It is saying the same thing. If you have one k32 then it is 101.4GiB versus 400 PiB. If your farm has 100 k32s then you have 9.9TiB versus 400 PiB of netspace today.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
